content/outcomes/cloud-native-development/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Cloud Native Development
+weight: -1
+layout: single
+description: "Learn the basics of cloud native development with Spring. This developer pre-engagement guide helps you to get the most of an engagement following the Tanzu Labs methodology."
+team:
+ - VMware Tanzu Labs
+---
+ 
+## Why is this learning path important?
+
+This learning path provides the foundational knowledge around modern, cloud-native software development with Spring and Cloud-Foundry. It has been conceived as pre-work to get the most of an application-related software development engagement with Tanzu Labs. 
+
+## Who is this learning path for?
+ 
+This learning path is for software engineers and other members of the technical development team involved in a Tanzu Labs engagement.
+
+## What you will learn
+ 
+Each section of this guide provides details about the skills or technologies in question, and includes links to specific and targeted information to assist you. Each section is followed by suggested homework assignments and additional resources.
+
+<<<<<<< HEAD
+It should take about eight hours to complete the material in this learning path. However, the more time you put into the learnng path, the more you will get out of the engagement.
+ 
+<<<<<<< HEAD
```

Comment:
Looks like we have some merge conflicts to resolve here..

Reply:
Good catch. There were merging issues in https://github.com/vmware-tanzu/tanzu-dev-portal/pull/997/commits/561b2d608e6cc5f36a662f7a4a741d4843f56b76 and https://github.com/vmware-tanzu/tanzu-dev-portal/pull/997/commits/27b5a9d105528e2f856c35a4b86207f0cc043eae

I reviewed @lizz7000 changes and fixed all issues in 961ee163a68737d1eb20b82c27aaa8537cdb402e